### PR TITLE
Only load status_config.php CSS when necessary

### PR DESCRIPTION
**Pull request by dregad**
As mentioned in https://github.com/mantisbt/mantisbt/pull/1831#issuecomment-1179776776

Fixes [#30773](https://mantisbt.org/bugs/view.php?id=30773)

### CODE REVIEW

File: lang/strings_english.txt [hunk 1]
```diff
@@ -178,6 +178,7 @@ $s_opensearch_text_description = '%s: full-text search';
 $s_select_option = '(select)';
 
 # bug_actiongroup_page.php : mass treatment
+$s_bug_actiongroup_result = 'The following issues could not be processed';
```
Comment (atrol) [inline lang/strings_english.txt:181]: This change is not related to loading status_config.php CSS

Comment (dregad) [reply lang/strings_english.txt:181]: You're right it belongs to another branch I was working on in parallel, it was committed by mistake.

**Comment by atrol** [comment]
@dregad do we still need this ugly special handling in status_config.php after your change?

```
if( $t_referer_page == auth_login_page() ) {
	# custom status colors not needed.
	http_caching_headers( false );
	exit;
}

switch( $t_referer_page ) {
	case AUTH_PAGE_USERNAME:
	case AUTH_PAGE_CREDENTIAL:
	case 'signup_page.php':
	case 'lost_pwd_page.php':
	case 'account_update.php':
		# We don't need custom status colors on login page, and this is
		# actually causing an error since we're not authenticated yet.
		http_caching_headers( false );
		exit;
}
```

**Review by vboctor — APPROVED** [review]
